content/courses/intro-to-embedded-sys-prog/chapters/low_level_programming.rst

```diff
@@ -1,38 +1,408 @@
 Low Level Programming
 =====================
 
+:code-config:`run_button=True;prove_button=False;accumulate_code=False`
```

Comment:
`:code-config:` isn't supported anymore — actually, one of tests is failing because of this. You can simply remove this line and the next one.

Reply:
Done